04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -185,10 +209,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<a class=\"anchor\" id=\"save\"></a>\n",
-    "### Saving results to virtual storage\n",
+    "<a class=\"anchor\" id=\"savevospace\"></a>\n",
+    "# Saving results to virtual storage VOSpace\n",
+    "\n",
+    "VOSpace is a convenient storage space for users to save their work. It can store any data or file type.  \n",
     "\n",
-    "Now we want to save the results from the same query to our virtual storage space instead.  By putting the query in a try-block we are able to trap errors when executing the query.  Note that running this cell multiple times will trigger an error and we use the Storage Manager client to remove the file once we are done."
+    "Now we want to save the results from the same query to our virtual storage space instead.  By putting the query in a try-block we are able to trap errors when executing the query.  Note that running this cell multiple times will trigger an error and we use the Storage Manager Client to remove the file once we are done."
```

Comment:
I think this should be maybe in 2-3 cells, and display that after running the query there is a new file in VOSpace?